### PR TITLE
Add `sqlserverflex instance create/update` commands

### PR DESCRIPTION
**Pull request by joaopalet**

### CODE REVIEW

File: internal/pkg/services/sqlserverflex/utils/utils.go
Comment (joaopalet) [inline internal/pkg/services/sqlserverflex/utils/utils.go:1]: Utils are similar to `mongodbflex`. Removed the ones that are still unused and adjusted some data types

File: internal/cmd/sqlserverflex/instance/update/update_test.go [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+)
+
+var projectIdFlag = globalflags.ProjectIdFlag
+
+type testCtxKey struct{}
+
+var testCtx = context.WithValue(context.Background(), testCtxKey{}, "foo")
+var testClient = &sqlserverflex.APIClient{}
+
+type mongoDBFlexClientMocked struct {
```
Comment (DiogoFerrao) [inline internal/cmd/sqlserverflex/instance/update/update_test.go:25]:
```suggestion
type SQLServerFlexClientMocked struct {
```

File: internal/cmd/sqlserverflex/instance/create/create_test.go [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+
+var projectIdFlag = globalflags.ProjectIdFlag
+
+type testCtxKey struct{}
+
+var testCtx = context.WithValue(context.Background(), testCtxKey{}, "foo")
+var testClient = &sqlserverflex.APIClient{}
+
+type mongoDBFlexClientMocked struct {
+	listFlavorsFails  bool
```
Comment (DiogoFerrao) [inline internal/cmd/sqlserverflex/instance/create/create_test.go:26]:
```suggestion
type SQLServerFlexClientMocked struct {
```

File: internal/cmd/sqlserverflex/instance/instance.go [hunk 3]
```diff
@@ -1,7 +1,9 @@
 package instance
 
 import (
+	"github.com/stackitcloud/stackit-cli/internal/cmd/argus/credentials/create"
```
Comment (DiogoFerrao) [inline internal/cmd/sqlserverflex/instance/instance.go:4]: Wrong import here 👀

File: internal/pkg/services/sqlserverflex/utils/utils_test.go [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+	testProjectId  = uuid.NewString()
+	testInstanceId = uuid.NewString()
+)
+
+const (
+	testInstanceName = "instance"
+	testUserName     = "user"
+)
+
+type mongoDBFlexClientMocked struct {
```
Comment (DiogoFerrao) [inline internal/pkg/services/sqlserverflex/utils/utils_test.go:25]:
```suggestion
type SQLServerFlexClientMocked struct {
```

File: internal/cmd/sqlserverflex/instance/instance.go (hunk 3 above)
Comment (joaopalet) [reply internal/cmd/sqlserverflex/instance/instance.go]: Great catch, I had to remove the command that I was using to etst before making the PR for the `list` and didn't notice in the re-import

**Pull request merged**